contracts/src/Allowlist.1.sol

```diff
@@ -2,7 +2,7 @@
 pragma solidity 0.8.20;
 
 import "./interfaces/IAllowlist.1.sol";
-import "./interfaces/IVersion.sol";
+import "./ProtocolVersion.sol";
 
 import "./libraries/LibAllowlistMasks.sol";
 import "./Initializable.sol";
```

Comment:
@iamsahu It seems this import is supposed to be ProtocolVersionV1 everywhere.

Reply:
The current import is correct since the file name is the same. We could have used the following format to do the import: `import {ProtocolVersionV1} from "./ProtocolVersion.sol";`
I didn't want to differ from rest of the imports so I have followed the same pattern as we have for the rest.

Reply:
Oh! I see, it's a bit confusing though. Why not change the file name?